internal/aez/power.go

```diff
@@ -18,6 +18,7 @@ type PWR struct {
 	PWRP3C3 uint16 // +3V3 current sense 0..4095
 }
 
-func (s *PWR) read(buf io.Reader) error {
-	return binary.Read(buf, binary.BigEndian, s)
+// Read PWR
+func (pwr *PWR) Read(buf io.Reader) error {
+	return binary.Read(buf, binary.BigEndian, pwr)
```

Comment:
how about a test of a real PWR buffer from a `.rac` file resulting in the expected PWR struct?